static/haven/example_3_person_invalid/server_response.json

```diff
@@ -0,0 +1,12 @@
+{
+  "eaid": "938920192837465",
+  "error": {
+    "type": "InvalidTaxHouseholdError",
+    "source_eapersonid": "0020",
+    "target_eapersonids": [
+      "0021",
+      "0022"
+    ],
+    "message": "Max Clark claims both to be a tax non-filer and sharing a tax return with Minnie Clark and Steve Clark. An individual may not be claimed as a filer if he or she does not file taxes."
```

Comment:
I suggest we not overcomplicate the error messages (unless this is a business requirement). Besides, note that if the first validation we do fails, we wouldn't obviously do the others. So, like we agreed upon for another similar use case, failing with a generic message would be a better way to go at this time.

In this case, for example, we wouldn't go as far as Steve Clark (if they are the last person) and simply fail since the "taxhousehold" attribute is incorrectly "Y" for the A/B relationship. So, we don't have that information to begin with.

In any case, if error messages (a) to this level of detail, and (b) a complete list beyond the first failure (as in, we keep going even if the first validation fails and collect a bag of validation issues), are both requirements, we can discuss them in more detail and track them via separate, specific cards.

Also, if we weren't to do that, I suggest we move all the error case into a sub directory (under /haven, etc).